.github/actions/setup-cli/action.yaml

```diff
@@ -22,14 +51,7 @@ runs:
 
     - id: setup-minikube
       run: |
-        ./hack/start-minikube.sh start --kubernetes-version=v${K8S_VERSION} --driver=$CONTAINER_RUNTIME --cpus $(nproc) --memory 5g
-      shell: bash
-
-    - id: docker-info
-      run: |
-        eval $(minikube docker-env)
-        docker ps
-        kubectl get nodes -o yaml
-        kubectl cluster-info
-        docker info
```

Comment:
this was displaying a few stat infos about docker engine and the cluster.

Reply:
Good catch, I've put it back but changed to `podman` thx @pedjak